SPECS/kata-containers/runtime-Support-for-AMD-SEV-SNP-VMs.patch

```diff
@@ -1,268 +0,0 @@
-From 22bda0838c77853ed92546308b59b6500083bb4e Mon Sep 17 00:00:00 2001
```

Comment:
https://github.com/kata-containers/kata-containers/commit/22bda0838c77853ed92546308b59b6500083bb4e merged in 3.1.0 
<img width="400" alt="image" src="https://user-images.githubusercontent.com/30801614/234128339-fa14a73b-a40c-46ec-8306-920572304c70.png">